packages/react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -117,7 +117,7 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
         aria-posinset={ariaPosInSet}
         data-is-focusable={true}
         onKeyUp={this._onKeyUp}
-        aria-label={group.ariaLabel || group.name}
+        aria-label={group.ariaLabel || `${group.name} (${group.count})`}
```

Comment:
I'm not sure, but I think a blank aria-label would be valid. In that case, this code would overlook the blank label and fall back.

Alternative solution:

```suggestion
        aria-label={group.ariaLabel !== undefined ? group.ariaLabel : `${group.name} (${group.count})`}
```

Reply:
Also... why not just provide your own aria-label?

Reply:
(That said I don't see this being that significantly worse than the original. And I just read through the original issue so i have more context.)